docs/grpc2.md

```diff
@@ -104,3 +104,49 @@ If these are enabled then the following options become available
   get_account_transaction_sign_hash = true
   get_block_items = true
   ```
+
+### Configuration options for checking client liveness
+
+The following configuration options for the GRPC2 server can be used to ensure
+connected clients are alive and responding in desired time.
+
+- `--grpc2-keep-alive-interval` (`CONCORDIUM_NODE_GRPC2_KEEPALIVE_INTERVAL`)
+  Enable HTTP2 keepalive, and set the interval (in seconds) at which Ping frames are sent. [env:
+
+- `--grpc2-keep-alive-timeout` (`CONCORDIUM_NODE_GRPC2_KEEPALIVE_TIMEOUT`)
+  Timeout (in seconds) for responses to HTTP2 Ping frames. If the client does not respond in time then the
+  connection will be closed. This has no effect unless `grpc2-keep-alive-interval` is set.
+
+- `--grpc2-tcp-keep-alive-interval` (`CONCORDIUM_NODE_GRPC2_TCP_KEEPALIVE`)
+  The interval (in seconds) at which TCP keepalive probes are sent.
+
+### Configuration options for limiting resource usage
+
+- `--grpc2-invoke-max-energy` (`CONCORDIUM_NODE_GRPC2_INVOKE_MAX_ENERGY`)
+  Maximum amount of energy allowed for the InvokeInstance and InvokeContract (V1 API) endpoints.
```

Comment:
The flag is named something like `xx_GRPC2_xx` however the comment states it's for V1 api?

Reply:
It applies to both. I'll revise the comment to make that more explicit.